Additional (optional) dependency

[AsciiDoc](https://asciidoc.org/) is an additional optional dependency not listed.

Output on Ubuntu 18.04 when compiling without a2x:

```
arch-install-scripts$ make
  GEN      arch-chroot
  GEN      genfstab
  GEN      pacstrap
  GEN      doc/arch-chroot.8
/bin/sh: 1: a2x: not found
Makefile:30: recipe for target 'doc/arch-chroot.8' failed
make: *** [doc/arch-chroot.8] Error 127
```

Output after installing `asciidoc-base`:

```
arch-install-scripts$ make
  GEN      arch-chroot
  GEN      genfstab
  GEN      pacstrap
  GEN      doc/arch-chroot.8
  GEN      doc/genfstab.8
  GEN      doc/pacstrap.8
```

--- a/README.md
+++ b/README.md
@@ -9,6 +9,7 @@ tasks when installing [Arch Linux](https://www.archlinux.org).
 * util-linux (>= 2.23)
 * POSIX awk
 * bash (>= 4.1)
+* asciidoc (for generating man pages)
 
 ## License
 

